### DISCUSSION TITLE
.ToBitmap(), .ToBitmapSource() in 7.19.0.1

### DISCUSSION THREAD

**Discussion by Yakonrus** [dlemstra/Magick.NET#666] [Help] [answered]
Hi.
.NET Framework 4.7.2
The latest working version is 7.17.0.1
7.19.0.1 - CS1061  C# does not contain a definition for and no accessible extension method accepting a first argument of type could be found (are you missing a using directive or an assembly reference?)

**Comment by dlemstra** [comment, marked as answer]
With a search you would have found #657 or #652.

**Comment by Yakonrus** [comment]
System.Drawing.Color color
using (MagickImage mask1 = new MagickImage()){
{
mask1.Transparent(color);
}
CS1503  C# Argument 1: cannot convert from 'System.Drawing.Color' to 'ImageMagick.IMagickColor<byte>'
How to convert?

**Comment by Yakonrus** [comment]
IMagickColor<byte> magickColor = new MagickColor();
                magickColor.SetFromColor(color);

**Comment by dlemstra** [comment]
You could do the following:

```C#
var magickColor = new MagickColor();
magickColor.SetFromColor(color);
mask1.Transparent(magickColor);
```

But I think it would be better to switch to using MagickColor instead.
